ReactiveCocoa/Swift/Signal.swift

```diff
@@ -211,20 +215,29 @@ public func filter<T, E>(predicate: T -> Bool) -> Signal<T, E> -> Signal<T, E> {
 			}
 		}
 	}
-}
 
-/// Unwraps non-`nil` values from `signal` and forwards them on the returned
-/// signal, `nil` values are dropped.
-public func ignoreNil<T, E>(signal: Signal<T?, E>) -> Signal<T, E> {
```

Comment:
This is one of those instances where attempting to constrain on an `OptionalType` would crash the compiler which is why I switched `filterMap`.